charts/amoro/templates/_pod.tpl

```diff
@@ -120,7 +120,7 @@ spark distribution package will be installed to here*/ -}}
 {{- if .Values.plugin.metricReporters }}
 {{- if .Values.plugin.metricReporters.prometheusExporter.enabled }}
 - name: prometheus
-  containerPort: {{ .Values.plugin.metricReporters.prometheusExporter.port }}
```

Comment:
if have anothor properties,not just only port.maybe can use map set of this config

Reply:
Thank you @czy006 , is this what you said?
 

    prometheusExporter:

      name: prometheus-exporter

      enabled: true

      port: 7001

      properties:

        port: {{ .Values.plugin.metricReporters.prometheusExporter.port }}